### PR TITLE
Add more example: simple page to fetch ng-query repository

### PR DESCRIPTION
**Pull request by trungk18**
## Description

This PR is to add more examples describe in #4 with a simple fetch to ng-query and show its repo information. 

More examples will be added once the corresponding API is ready e.g pagination, mutation etc.

Technically we should follow what are the original examples from react-query so that users know what to expect.

## Screenshots

image

## Question 

Are we plan to support normal `@Inject(QueryProvider) private useQuery: QueryType`? 
If so, we will need to export the `Query['query']`, not sure what to call it anyway 😂

image

## Follow up

We should update the repo description. Something like that? 

```
Angular wrapper for @tanstack/query 🔥 Powerful asynchronous state management, server-state utilities and data fetching library.
```

image

### CODE REVIEW

File: packages/playground/src/app/simple-page/simple-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <strong>👀 {{repo.data.subscribers_count }}</strong>
+        <strong>✨ {{repo.data.stargazers_count }}</strong>
+        <strong>🍴 {{repo.data.forks_count }}</strong>
+        <div>{{repo.isFetching ? "Updating..." :  ""}}</div>
+      </div>
+    </ng-container>
+  `,
+})
+export class SimplePageComponent {
+  useQuery = inject(QueryProvider);
```
Comment (trungk18) [inline packages/playground/src/app/simple-page/simple-page.component.ts:35]: @NetanelBasal Is this OK to `inject` within the component or do you prefer to move it into a service?

Comment (NetanelBasal) [reply packages/playground/src/app/simple-page/simple-page.component.ts:35]: Our examples should show the best practices, so let's always user services.

Comment (trungk18) [reply packages/playground/src/app/simple-page/simple-page.component.ts:35]: Sure, I'll update the code.

**Comment by NetanelBasal** [comment]
- You can add the description in our README for now. We'll deal with the README when we release our beta. 
- I prefer not using the param decorator because it's not part of the JS spec. If users want a different way, we'll expose the service.

**Comment by trungk18** [comment]
@NetanelBasal Sure, I will look into our API later and then add more examples.

**Pull request merged**